docs/pages/eas-update/asset-selection.mdx

```diff
@@ -28,15 +29,35 @@ To use this feature, include the new property `extra.updates.assetPatternsToBeBu
 
 In this case, all **.png** files in all subdirectories of **app/images** will be included in updates.
 
-Assets that do not match any of the file patterns will still resolve in the Metro bundler, but will not be uploaded to the updates server. If making use of this feature, a developer needs to be certain that assets not included in updates are built into the native build of the app.
-
 If this new property is not in your app config, all assets resolved by the bundler will be included in updates (as per SDK 49 and earlier behavior).
 
+## Verifying that an update includes all required app assets
+
+When using this feature, assets that do not match any file patterns will resolve in the Metro bundler. However, these assets will not be uploaded to the updates server. You have to be certain that assets not included in updates are built into the native build of the app.
+
+If you are building your app locally or have access to the correct build for publishing updates (with the same [runtime version](/eas-update/runtime-versions/)), then you can use the `npx expo-updates assets:verify` command. It allows you to check whether all required assets will be included when you publish an update:
+
+> **info** This new command is part of the `expo-updates` CLI, which also supports [EAS Update code signing](/eas-update/code-signing/). It is not part of the [Expo CLI](/more/expo-cli/). It is available for ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.24.10).
```

Comment:
Because most people who run EAS Update will be using eas-cli
```suggestion
> **info** This new command is part of the `expo-updates` CLI, which also supports [EAS Update code signing](/eas-update/code-signing/). It is not part of the [Expo CLI](/more/expo-cli/) or [eas-cli](https://github.com/expo/eas-cli). It is available for ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.24.10).
```